src/documentation/Usage/UsageCss.stories.mdx

```diff
@@ -11,66 +11,95 @@ import { linkTo } from '@storybook/addon-links';
 
-Make sure you have a scss loader installed.
+## Usage with vanilla JS
 
-#### Import scss via react
+The JS components are built using React, but that **does not limit their usage
+to just React applications**. You can render any component in any JavaScript
+application with `ReactDOM.render`. Think of it like using a jQuery plugin.
+Make sure you have embedded the CSS of the UI Kit in your project.
 
-```
-// Import all stylesheets
-import '@wfp/ui/src/globals/scss/styles.scss';
+```js
+import ReactDOM from 'react-dom';
+import { Button } from '@wfp/ui';
 
-// Only import the Button stylesheet
-import '@wfp/ui/src/components/Button/button.scss';
+const myContainerElement = document.getElementById('container');
+
+// with JSX
+ReactDOM.render(<Button>Simple Button</Button>, myContainerElement);
 
+// with vanilla JS, use React.createElement
+ReactDOM.render(
+  React.createElement(Button, {
+    className: Classes.SMALL,
+    children: 'Simple button',
+  }),
+  myContainerElement
+);
+```
+To remove the component from the DOM and clean up, unmount it:
+```
+ReactDOM.unmountComponentAtNode(myContainerElement);
 ```
 
-### SCSS
+Check out the [React API docs](https://reactjs.org/docs/react-api.html) for more details.
 
-When using scss you can import the SCSS sources from the node_module. This allows you to use only parts of the SCSS, reuse common variables and functions from WFP UI or to override settings.
+## Usage with pure HTML
+**You can grab the HTML of every component or template** directly from these docs: simply navigate to the item you are looking for, search for the in-page preview under the "Documentation" tab and click on the HTML tab. For the styling, you can [load the CSS through the WFP CDN](https://uikit.wfp.org/docs/index.html?path=/docs/getting-started-installing-the-kit--page#3-load-the-css). We are currently evaluating to expose the CSS for the single components and templates.
 
-#### Use Source scss
+## Using the Kit SCSS
 
-```
+When using scss you can import the SCSS sources from the node_module.
+This allows you to use only parts of the SCSS, reuse common variables
+and functions from WFP UI or to override settings.
+
+```css
 // Sample overriding settings
 $input-border-radius: 5px;
 $button-border-radius: 60px;
 $button-font-size: 25px;
 $button-padding: 10px 25px;
-$color__main: #0076FF;
-
-// Import all stylesheets
-@import "~@wfp/ui/src/globals/scss/styles.scss";
+// new css variables used for colours
+--interactive-01: #0076FF;
 
-// Only import the Button stylesheet
-@import "~@wfp/ui/src/components/Button/button.scss";
+// Disable css reset
+$css--reset==false
 
 // Only use variables
 @import "~@wfp/ui/src/globals/scss/vars.scss";
 ```
 
-### CDN Usage
-
-Import the WFP UI css from the WFP CDN. Change the version number to the UI Kit version you are using.
-
-#### CDN
-
-```
-<link
-  rel="stylesheet"
-  type="text/css"
-  href="https://cdn.wfp.org/guides/ui/v1.8.2/assets/css/styles.min.css"
-/>
-```
+***
```

Comment:
Let's make the `Help us improve` message globally available.

Reply:
Yep, there is already a task in the backlog for that for @cassandrakesewa